src/lib/utils/thorchain/balance.ts

```diff
@@ -112,7 +112,12 @@ export const fetchHasEnoughBalanceForTxPlusFeesPlusSweep = async ({
 
         return queryClient.fetchQuery({
           queryKey: thorchainSaversWithdrawQuoteQueryKey,
-          queryFn: getThorchainSaversWithdrawQuoteQueryFn,
+          queryFn: () =>
+            getThorchainSaversWithdrawQuoteQueryFn({
+              asset,
+              accountId,
+              amountCryptoBaseUnit: withdrawAmountCryptoBaseUnit,
+            }),
```

Comment:
Note, with the takeaways from react-query type-safety, we're not using a magic `queryFn` which leverages the queryKey, but a good ol' async fn which is type-safe